Description: Fixed namespaced oslo imports
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-02-08

--- python-oslo.serialization-1.2.0.orig/tests/test_jsonutils.py
+++ python-oslo.serialization-1.2.0/tests/test_jsonutils.py
@@ -24,8 +24,8 @@ import simplejson
 import six
 import six.moves.xmlrpc_client as xmlrpclib
 
-from oslo.i18n import fixture
-from oslo.serialization import jsonutils
+from oslo_i18n import fixture
+from oslo_serialization import jsonutils
 
 
 class JSONUtilsTestMixin(object):
--- python-oslo.serialization-1.2.0.orig/oslo_serialization/jsonutils.py
+++ python-oslo.serialization-1.2.0/oslo_serialization/jsonutils.py
@@ -59,9 +59,9 @@ else:
 import six
 import six.moves.xmlrpc_client as xmlrpclib
 
-from oslo.utils import encodeutils
-from oslo.utils import importutils
-from oslo.utils import timeutils
+from oslo_utils import encodeutils
+from oslo_utils import importutils
+from oslo_utils import timeutils
 
 netaddr = importutils.try_import("netaddr")
 
--- python-oslo.serialization-1.2.0.orig/oslo_serialization/tests/test_jsonutils.py
+++ python-oslo.serialization-1.2.0/oslo_serialization/tests/test_jsonutils.py
@@ -24,7 +24,7 @@ import simplejson
 import six
 import six.moves.xmlrpc_client as xmlrpclib
 
-from oslo.i18n import fixture
+from oslo_i18n import fixture
 from oslo_serialization import jsonutils
 
 
